ios/AirMaps/AIRMapUrlTile.h

```diff
@@ -0,0 +1,35 @@
+//
+//  AIRUrlTileOverlay.h
+//  AirMaps
+//
+//  Created by cascadian on 3/19/16.
+//  Copyright © 2016. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <MapKit/MapKit.h>
+#import <UIKit/UIKit.h>
+
+#import "RCTConvert+MapKit.h"
+#import "RCTComponent.h"
+#import "AIRMapCoordinate.h"
+#import "AIRMap.h"
+#import "RCTView.h"
+
+@interface AIRMapUrlTile : MKAnnotationView <MKOverlay>
```

Comment:
I would love to get @lelandrichardson's thoughts here: I think this PR is following a pattern established with other views, but it feels like we have an unnecessary layer of indirection here.

Instead of `AIRMapUrlTile` being a subclass of `MKAnnotationView` with a `MKTileOverlay` property, couldn't we instead make `AIRMapUrlTile` a subclass of `MKTileOverlay`?

The primary issue I see with this is in `AIRMapUrlTileManager.m`, where we initialize the `AIRMapUrlTile` instance with no parameters and then later apply the value of `urlTemplate` from RN. If `AIRMapUrlTile` were a subclass of `MKTileOverlay` we would need access to `urlTemplate` at `init` time. It would also complicate things if `urlTemplate` were to change.

What do you think @lelandrichardson?